app/models/roles_metadata_delegate_regions.rb

```diff
@@ -5,7 +5,7 @@ class RolesMetadataDelegateRegions < ApplicationRecord
     senior_delegate: "senior_delegate",
     regional_delegate: "regional_delegate",
     delegate: "delegate",
-    junior_delegate: "candidate_delegate",
+    candidate_delegate: "candidate_delegate",
```

Comment:
Veto. The official terminology is "Junior Delegate", so you should change the actual stored values rather than changing the key

Reply:
(This also implies changes throughout the rest of the diff of the whole PR, that I will not point out one by one)

Reply:
Thing is, the value in `users` table currently is `candidate_delegate` and is used in many places. Using the key `junior_delegate` here is not working because the value stored in database is `candidate_delegate`. My plan is to do the change all occurrences of candidate_delegate to junior_delegate in a later PR because it will involve much more files. Is that fine?

Reply:
Okay, in that case you can just let the value be `"candidate_delegate"` but still keep the _key_ at `junior_delegate` (as it already has been before submitting this PR).

In other words: Why do you need to change the _key_ here? Rails enums feature is exactly the solution that should allow you to keep the _value_ in the database intact, but use a consistent _key_ in the code...